tests/smoke.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+TESTS="pwd, lsb_release -a, uname -a, ip a, lscpu, vmstat, lsblk, false"
+LOGS="/tmp/TESTS_$(date +%s)_"
```

Comment:
Unused variable

Reply:
I wanted to catch the output of each test, but I'll remove it for now. Catching exit status and log at the same time requires sth like this: https://github.com/Linaro/test-definitions/blob/master/automated/lib/sh-test-lib#L322. I don't think we need it right now.